adapters/flipp/flipp.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/url"
 	"strings"
 
+	"github.com/SirDataFR/iabtcfv2"
```

Comment:
@hasan-kanjee could you help to understand usage of `iabtcfv2` package in your adapter code

Reply:
The purpose of that package is so that I can decode a user consent string to determine whether TCF Purpose 4 is allowed or not. I am not sure if there is any other way to decode the user consent string? You can see how I am using it here https://github.com/prebid/prebid-server/blob/f8a96e294e1898cee1b2df5f96d4d61748f39c32/adapters/flipp/flipp.go#L276

Reply:
@hasan-kanjee  thanks for the explanation. Prebid recommends to use https://github.com/prebid/go-gdpr package for decoding TCF string

Reply:
@onkarvhanumante fixed! Using https://github.com/prebid/go-gdpr instead